tests/client/test_client_task_meilisearch.py

```diff
@@ -163,7 +162,3 @@ def test_delete_all_tasks(client):
     assert task.type == "taskDeletion"
     assert len(tasks_after["results"]) >= 1
     assert len(tasks_before["results"]) == len(tasks_after["results"])
```

Comment:
`assert len(tasks_before["results"]) == len(tasks_after["results"])` should be removed. The `assert` that should stay is

```py
    assert (
        "statuses=succeeded%2Cfailed%2Ccanceled"
        in tasks_after["results"][0]["details"]["originalFilter"]
    )
```

Reply:
I have done the suggested changes, can you look into it again?